myproject/myproject/urls.py

```diff
@@ -5,4 +5,5 @@
     path('admin/', admin.site.urls),
     path('blog/', include("blog.urls")),
     path('users/', include("users.urls")),
-]
+    path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
Add one variable in settings called `DEBUG_TOOLBAR`. Set the value to 0 initially. And add the condition here like if DEBUG_TOOLBAR is set to 1, then only shows this URL because we use this url only for profile, so we need that locally to profile the code and queries.